prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -433,7 +433,7 @@ class FHIRRouter(
             if (isDefaultFilter(filterType, filters)) "(default filter) "
             else ""
             }${failingFilterName ?: "unknown"}"
-            logFilterResults(filterToLog, bundle, report, receiver, filterType)
+            logFilterResults(filterToLog, bundle, report, receiver, filterType, focusResource)
```

Comment:
Existing unit tests for `logFilterResults` are no longer passing with this pattern change.